utils/make-pkgconfig.swift

```diff
@@ -46,24 +46,21 @@ extension String: Error {
 }
 
 func makeFile() throws {
-  let pkgConfigPath = "/usr/local/lib/pkgconfig"
+  guard let brew = which("brew") else { throw "Missing callable `brew` instance." }
+  guard let brewPrefix = run(brew, args: ["--prefix"]) else { throw "Brew could not detect installation prefix location." }
+  let pkgConfigPath = "\(brewPrefix)/lib/pkgconfig"
   let pkgConfigDir = URL(fileURLWithPath: pkgConfigPath)
 
-  // Make /usr/local/lib/pkgconfig if it doesn't already exist
+  // Make <brew-prefix>/lib/pkgconfig if it doesn't already exist
   if !FileManager.default.fileExists(atPath: pkgConfigPath) {
     try FileManager.default.createDirectory(at: pkgConfigDir,
                                             withIntermediateDirectories: true)
   }
   let cllvmPath = pkgConfigDir.appendingPathComponent("cllvm.pc")
-
-  let brewLLVMConfig: () -> String? = {
-    guard let brew = which("brew") else { return nil }
-    guard let brewPrefix = run(brew, args: ["--prefix"]) else { return nil }
-    return which(brewPrefix + "/opt/llvm/bin/llvm-config")
-  }
+  let brewLLVMConfig = which(brewPrefix + "/opt/llvm/bin/llvm-config")
```

Comment:
Let's keep this in a closure since it only needs to be executed if the other 2 `which` calls on line 63 returned `nil`.

Reply:
Okay